src/app/detail/ui/ActivityWrapper.tsx

```diff
@@ -0,0 +1,70 @@
+'use client'
+
+import { LogResponse } from '@/app/api/activity-logs/schemas/LogResponseSchema'
+import { Box, Skeleton, Stack, Typography } from '@mui/material'
+import { Comments } from './Comments'
+import { ActivityLog } from './ActivityLog'
+import { CommentInput } from '@/components/inputs/CommentInput'
+import { ActivityType } from '@prisma/client'
+import { postComment, deleteComment } from '../[task_id]/[user_type]/actions'
+import { CreateComment } from '@/types/dto/comment.dto'
+import { fetcher } from '@/utils/fetcher'
+import useSWR from 'swr'
+
+export const ActivityWrapper = ({ token, task_id }: { token: string; task_id: string }) => {
+  const { data: activities, isLoading } = useSWR(`/api/tasks/${task_id}/activity-logs/?token=${token}`, fetcher, {
+    refreshInterval: 1000,
```

Comment:
1 seconds is too low for a polling implementation. There will be multiple concurrent requests on top of each other if the requests are not returned within 1 second.
Something like 10-20 seconds is more viable

Reply:
The purpose to set 1 second polling interval time is because it is strictly mentioned in the ticket that it should be nearly realtime "Remember that this has to be near real-time", and while using polling, this is the best way to achieving near real time by increasing the amount of polling.